kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/repair/KafkaTaskFailureListener.java

```diff
@@ -11,9 +11,6 @@
 
 import java.util.Arrays;
 
-/**
- * Created by gabriel on 8/8/16.
- */
 public class KafkaTaskFailureListener implements TaskFailureListener {
```

Comment:
The "Created by gabriel" docs are bad.  Can you replace it with something better?  Mostly copied from TaskFailureListener interface is fine.